nano/node/common.cpp

```diff
@@ -696,6 +696,21 @@ void nano::keepalive::serialize (nano::stream & stream_a) const
 	}
 }
 
+std::string nano::keepalive::to_string ()
+{
+	std::stringstream stream;
+
+	stream << "\n" + header.to_string () + "\n";
```

Comment:
Why is there a leading new line? That looks weird. It should be up to the caller to put leading or trailing whitespace. This code should use minimal whitespace.

Reply:
done